pkg/api/instance.go

```diff
@@ -0,0 +1,17 @@
+package api
+
+type Instance struct {
```

Comment:
comments here?

Reply:
added comments to clarify that the instance API object is internally utilized by Maestro for discovering its peers.

Reply:
"Instance" isn't a meaningful name.  what is an "instance"?

Reply:
What about "maestroInstance"? do you think it is slightly redundant?